cmd/gf/internal/cmd/cmd_gen_dao.go

```diff
@@ -1,6 +1,7 @@
 package cmd
 
 import (
+	_ "github.com/gogf/gf/contrib/drivers/clickhouse/v2"
```

Comment:
Try to generate code after introducing the clickhouse-go driver. An error may occur during direct generation.

Reply:
We used clickhouse for data storage in our new project. when I executed `gf gen dao` command, no new files were generated, as a new framework users of gf which was very confusing for me.

Reply:
If there are a lot of changes to solve this issue, please close this pull request